app/frontend/src/styles/application/jobseekers/jobseeker.scss

```diff
@@ -20,5 +20,17 @@
     .moj-filter-layout__content {
       overflow-x: hidden;
     }
+
+    .govuk-form-group {
```

Comment:
This feels quite broad. We can set classes on the specific form group by passing `form_group: { classes: "govuk-!-margin-0" }` to the form builder input for example

Reply:
the style wasnt actually needed, the nesting of the markup was a little off